addons/website_event_questions/static/tests/tours/tickets_questions.js

```diff
@@ -49,6 +49,10 @@ tour.register('test_tickets_questions', {
 }, {
     trigger: 'button:contains("Continue")',
     run: 'click'
+},{
```

Comment:
`}, {`

Also, I'd add a brief comment in the code to explain why you need this step.

Besides that, I understand and agree with the change.